feature-staking-impl/src/main/java/io/novafoundation/nova/feature_staking_impl/domain/StakingInteractorExt.kt

```diff
@@ -71,11 +72,13 @@ fun minimumStake(
 
     val stakeByNominator = exposures
         .fold(mutableMapOf<AccountIdKey, BigInteger>()) { acc, exposure ->
-            exposure.others.forEach { individualExposure ->
-                val key = individualExposure.who.intoKey()
-                val currentExposure = acc.getOrDefault(key, BigInteger.ZERO)
-                acc[key] = currentExposure + individualExposure.value
-            }
+            exposure.others
+                .take(maxNominatorsInValidator)
```

Comment:
Shouldn't `exposure.others` be sorted before taking first `maxNominatorsInValidator`?

Reply:
We get nominators with nice sort from on chain so I think it's not necessary